inference-engine/tests/functional/inference_engine/transformations/algebraic_simplification.cpp

```diff
@@ -35,10 +35,10 @@ TEST(algebraic_simplification, add_negative_tests) {
     auto c = make_shared<op::Parameter>(type, shape);
     auto abs_a = make_shared<op::Abs>(a);
     auto iconst2 = ngraph::make_constant_from_string("2", type, shape);
-    auto add_a_0 = a + iconst2;
```

Comment:
I suppose this tests should be deleted at all since as far as I understand an original goal of them was to test overloaded math operators